app/src/test/Sandbox.ts

```diff
@@ -31,6 +31,7 @@ export class Sandbox {
         });
         this.mounted = [];
         alt.recycle();
+        resetFetcher();
```

Comment:
I was getting some spurious test failures where other tests had left the fetcher in a weird state. The problem with global state! This helps, as any test that uses the sandbox (which is most of them) will then clear up after itself.